cmake/ngraph.cmake

```diff
@@ -28,6 +28,21 @@ list(APPEND NGRAPH_EXTRA_CMAKE_FLAGS "-DNGRAPH_DEX_ONLY=1")
 list(APPEND NGRAPH_EXTRA_CMAKE_FLAGS "-DNGRAPH_UNIT_TEST_ENABLE=0")
 list(APPEND NGRAPH_EXTRA_CMAKE_FLAGS "-DNGRAPH_TOOLS_ENABLE=0")
 
+if (MKLDNN_INCLUDE_DIR AND MKLDNN_LIB_DIR)
+  list(APPEND NGRAPH_EXTRA_CMAKE_FLAGS "-DMKLDNN_INCLUDE_DIR=${MKLDNN_INCLUDE_DIR}")
+  list(APPEND NGRAPH_EXTRA_CMAKE_FLAGS "-DMKLDNN_LIB_DIR=${MKLDNN_LIB_DIR}")
+  MESSAGE(STATUS "nGraph will use the MKLDNN library provided by MXnet")
+  MESSAGE(STATUS "   MKLDNN_INCLUDE_DIR='${MKLDNN_INCLUDE_DIR}'")
+  MESSAGE(STATUS "   MKLDNN_LIB_DIR='${MKLDNN_LIB_DIR}'")
+elseif(MKLDNN_INCLUDE_DIR OR MKLDNN_LIB_DIR)
+  MESSAGE(WARNING
+"Just one of MKLDNN_INCLUDE_DIR or MKLDNN_LIB_DIR is set, so \
+nGraph will build its own instance of libmkldnn.")
```

Comment:
The lack of indentation is due to CMake's ugly rules regarding long strings.  The alternative is to have a wide, single-line statement here.